pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2553,6 +2561,35 @@ func handlePhysicalIOAdapterListImpl(ctxArg interface{}, key string,
 			// We assume AddOrUpdateIoBundle will preserve any
 			// existing IsPort/IsPCIBack/UsedByUUID
 			aa.AddOrUpdateIoBundle(log, *ib)
+
+			// Adding PF before VF to have correct boot order
+			if ib.Type == types.PhyIoNetEthPF && ib.Vfs.Count > 0 {
+				ifNameMap, err := nettb.GetPciToIfNameMapByTimeout(60 * time.Second)
```

Comment:
You are reading all of them, but you only care about the ifname for this particular pcilong. We already have code which does that in EVE-OS, hence no neefd for an additional package.

Reply:
Where do we have this?

Reply:
PciLongToIfname() in types/ifnametopci.go

Reply:
Found it, thanks a lot! Will change and remove package

Reply:
Done